src-docs/src/views/elastic_charts/accessibility_example.js

```diff
@@ -0,0 +1,313 @@
+import React from 'react';
+import {
+  EuiCallOut,
+  EuiCode,
+  EuiCodeBlock,
+  EuiSpacer,
+  EuiText,
+} from '../../../../src/components';
+import { GuideSectionTypes } from '../../components';
+import { ExternalBadge } from './shared';
+import { TextureMultiSeriesChart } from './texture';
+import { TexturedStylesProps, ChartProps } from './texture_props';
+
+const TextureMultiSeriesChartSource = require('!!raw-loader!./texture');
+
+export const ElasticChartsAccessibilityExample = {
+  title: 'Accessibility features',
+  intro: (
+    <>
+      <ExternalBadge />
+      <EuiSpacer size="l" />
+      <EuiText>
+        <p>
+          Elastic charts is becoming more and more accessible for all users.
+        </p>
+        <p>
+          <strong>Available a11y-related features</strong>
+        </p>
+        <ul>
+          <li>Chart titles</li>
+          <li>Chart descriptions</li>
+          <li>
+            Chart data in tabular form<sup>*</sup>
+          </li>
+          <li>
+            Texture fill<sup>*</sup>
+          </li>
+          <li>
+            Semantic groupings<sup>*</sup>
+          </li>
+        </ul>
+        <p>
+          <small>
+            <sup>*</sup>Not all chart types that may benefit from this feature
+            have it implemented as of yet
+          </small>
+        </p>
+      </EuiText>
+    </>
+  ),
+  sections: [
+    {
+      title: 'Titles',
+      text: (
+        <>
+          <p>
+            <EuiCode>{'ariaLabel'}</EuiCode> and{' '}
+            <EuiCode>{'ariaLabelledBy'}</EuiCode> props can be set through the{' '}
+            <EuiCode>{'<Settings />'}</EuiCode> component.
+            <EuiSpacer />
+            <EuiCode>{'ariaLabel'}</EuiCode> takes a string label which will be
+            visually hidden whereas the
+            <EuiCode>{'ariaLabelledBy'}</EuiCode> prop can tie the chart to
+            other accessible elements elsewhere in your app.
+          </p>
+          <EuiCodeBlock language="jsx" isCopyable fontSize="s">
+            {`<Fragment>
+  <h1 id="randomHeadingId">Days of the week: weekdays vs weekends</h1>
+  <Chart>
+    <Settings
+      ariaLabelledby="randomHeadingId"
+    />
+    <BarSeries {...props} />
+  </Chart>
+</Fragment>`}
+          </EuiCodeBlock>
+          <p>
```

Comment:
Instead of using `<EuiSpacer />` between blocks of text, wrapping everything in `<p>` tags sets up the right spacing with more semantics too